tests/test_view_single_result.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.resultset import ResultsetPage
+
+
+class TestSingleResult:
+
+    @pytest.mark.nondestructive
+    def test_open_single_result(self, base_url, selenium):
+        # Select single test result and verify the content
+        resultset_page = ResultsetPage(base_url, selenium).open()
+
+        resultsetnumber = resultset_page.get_first_resultset_number()
+        resultset_page.open_single_resultset()
+
+        assert resultset_page.get_url_current_page().endswith(resultsetnumber)
```

Comment:
I don't think we need to abstract the Selenium property for the current URL. This can simply be `assert selenium.current_url.endswith(revision)`